web_registry/src/stores/RootStore.ts

```diff
@@ -14,6 +14,7 @@ export interface IRootStore {
 
     // Helpers
     getAssessmentContent: (assessmentId: string) => IAssessmentContent | undefined;
+    getLifeAreaContent: (lifeArea: string) => ILifeAreaContent | undefined;
```

Comment:
It seems this is a `lifeAreaId` and should be named that for clarity?

Reply:
Done